runtime/oti/j2sever.h

```diff
@@ -84,9 +84,7 @@
 	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_V12
 #endif
 #define J2SE_SHAPE_OPENJDK 		0x10000
-#define J2SE_SHAPE_B136    		0x40000
-#define J2SE_SHAPE_B148    		0x50000
-#define J2SE_SHAPE_B165    		0x60000
+#define J2SE_SHAPE_V9           0x60000
```

Comment:
This one, which uses spaces, doesn't line up in the github view with the others, which use tabs. Also, they could be renumbered, although it could also be done when adding V13 support.

Reply:
IMHO, there should be no tabs in any source file except, possibly, a group at the beginning of a line. We can fix the others when we get to Java 13 support (or remove the notion of 'shape' altogether - it seems mostly redundant now).